ndsl/grid/generation.py

```diff
@@ -297,12 +297,34 @@ def __init__(
         self._dy_center = None
         self._area = None
         self._area_c = None
-        (
-            self._ks,
-            self._ptop,
-            self._ak,
-            self._bk,
-        ) = self._set_hybrid_pressure_coefficients(eta_file, ak, bk)
+        if eta_file != "":
```

Comment:
Check `eta_file is not None`

Reply:
I had tried that initially, but the linting crashed.

Reply:
Pushed